kinisi/parser.py

```diff
@@ -574,11 +563,15 @@ def get_indices(structure: "MDAnalysis.universe.Universe", specie: str) -> Tuple
         """
         indices = []
         framework_indices = []
+
         if not isinstance(specie, list):
             specie = [specie]
+
         for i, site in enumerate(structure):
             if site.type in specie:
                 indices.append(i)
+            elif i in specie:
```

Comment:
So this means that we can pass a mixed list? 

What happens if the `specie` is like `'1'` (I recognise that the test you have included covers this case, but I am thinking from the users point of view)? Do we want to have a different keyword arg? (genuine question looking for thoughts on).